pkg/webhooks/operator_webhooks.go

```diff
@@ -147,6 +173,65 @@ func (webhookConfig *IntegreatlyWebhookConfig) Reconcile(ctx context.Context, cl
 	return nil
 }
 
+// ReconcileService creates or updates the service that points to the Pod
+func (webhookConfig *IntegreatlyWebhookConfig) ReconcileService(ctx context.Context, client k8sclient.Client) error {
+	service := &corev1.Service{
```

Comment:
@sergioifg94 should this have an owner ref to rhmi cr? I notice on previous installations that this doesn't get cleaned up when rhmi is uninstalled.

Reply:
+1, nice catch 👍

Reply:
I didn't know about this owner ref feature, I'll give it a look. Thanks!